src/pybind/mgr/dashboard/controllers/public.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import re
+
+from . import ApiController, BaseController, Endpoint
+from .. import mgr
+
+
+@ApiController('/public', secure=False)
+class Public(BaseController):
+    @Endpoint('GET')
+    def version(self):
+        raw_version = mgr.version
```

Comment:
As Lenz mentioned the version is actually provided in the `/summary` endpoint and the parsing of the different elements is done on the Front-end side by means of a pipe in the html template (pipes are in `src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/`).